datacenter/dtr/2.2/guides/admin/monitor-and-troubleshoot/troubleshoot-with-logs.md

```diff
@@ -51,27 +51,44 @@ commands:
 
 {% raw %}
 ```bash
-# REPLICA_ID will be the replica ID for the current node.
-REPLICA_ID=$(docker ps -lf name='^/dtr-rethinkdb-.{12}$' --format '{{.Names}}' | cut -d- -f3)
+# DTR_REPLICA_ID will be the replica ID for the current node.
+DTR_REPLICA_ID=$(docker ps -lf name='^/dtr-rethinkdb-.{12}$' --format '{{.Names}}' | cut -d- -f3)
+# List problems in the cluster detected by the current node.
+echo 'r.db("rethinkdb").table("current_issues")' | \
+  docker run -i --rm \
+    --net dtr-ol \
+    -e DTR_REPLICA_ID=${DTR_REPLICA_ID} \
+    -v dtr-ca-$DTR_REPLICA_ID:/ca \
+    dockerhubenterprise/rethinkcli:v2.2.0-ni non-interactive; \
+    echo
+```
+{% endraw %}
+
+On a healthy cluster the output will be `[]`.
+
+RethinkDB stores data in different databases that contain multiple tables. This
+container can also be used to connect to the local DTR replica and
+interactively query the contents of the DB.
+
+{% raw %}
+```bash
+# DTR_REPLICA_ID will be the replica ID for the current node.
+DTR_REPLICA_ID=$(docker ps -lf name='^/dtr-rethinkdb-.{12}$' --format '{{.Names}}' | cut -d- -f3)
 # This command will start a RethinkDB client attached to the database
 # on the current node.
 docker run -it --rm \
   --net dtr-ol \
-  -v dtr-ca-$REPLICA_ID:/ca dockerhubenterprise/rethinkcli:v2.2.0 \
-  $REPLICA_ID
+  -e DTR_REPLICA_ID=${DTR_REPLICA_ID} \
+  -v dtr-ca-$DTR_REPLICA_ID:/ca \
+  dockerhubenterprise/rethinkcli:v2.2.0-ni
```

Comment:
Make sure that were using the `dockerhubenterprise/rethinkcli:v2.2.0` non-ni image here, the `-ni` image won't send users to an interactive prompt iirc.

Preferably, I'd just tell users to use `v2.3.0` since that's the latest `rethinkcli`.

Reply:
```
$ docker run -it --rm \
>   --net dtr-ol \
>   -e DTR_REPLICA_ID=${DTR_REPLICA_ID} \
>   -v dtr-ca-$DTR_REPLICA_ID:/ca \
>   dockerhubenterprise/rethinkcli:v2.2.0-ni
Unable to find image 'dockerhubenterprise/rethinkcli:v2.2.0-ni' locally
v2.2.0-ni: Pulling from dockerhubenterprise/rethinkcli
85b1f47fba49: Pull complete
ba6bd283713a: Pull complete
b9968e24de01: Pull complete
838ee1f471db: Pull complete
0aeceafac5a9: Pull complete
ac225a9df894: Pull complete
7f1e665a07d9: Pull complete
8ad9826305ed: Pull complete
Digest: sha256:f09c5557fb44a5c9f344b240a2fe9df58b43f93910d61cfcd86b1b6ebf8cf232                                                                              
Status: Downloaded newer image for dockerhubenterprise/rethinkcli:v2.2.0-ni
> 
```

if you don't give it `non-interactive` as the first argument, `v2.2.0-ni` will still give you a prompt, and seems to format output similarly to the non-ni variant.    Would like to try and stay consistent with respect to the image tag used between examples.

- does `v2.3.0` have a non-interactive mode?  poked it a few ways and couldn't get it to do non-interactive.
- what's better about 2.3.0 for interactive mode?

Reply:
TIL.  There's a few bugs that have been fixed in 2.3.0, but I haven't noticed any major improvements, so whatever.

Reply:
@trapier Works for me then, I guess just make modifications to the latest 2.5.4 docs to support the use `dockerhubenterprise/rethinkcli` there instead of `docker exec` and we're golden.  Note that the non-interactive rethinkcli isn't packaged in the offline install of DTR in those versions, but that's kind of the same situation we were in in older versions of DTR anyways.

I'm discussing that with DTR to readd non-interactivity back into rethinkcli in future versions of the cli image.

Reply:
Can you make the changes to 2.5.4 regarding exec in a follow-up PR?